cs/arrays.texy

```diff
@@ -230,4 +228,243 @@ $res = Arrays::map($array, function($value, $key, $arr) { return $v . $v; });
 \--
 
 
+ArrayHash
+=========
+[Nette\Utils\ArrayHash |api:] je třída umožňující pracovat s polem jako objektem.
+
```

Comment:
Co třeba: Objekt ArrayHash je potomkem anonymního objektu stdClass, který jej rozšiřuje jeho o schopnost přistupovat ke členům, jako by to bylo pole, tedy přes hranaté závorky. Implementuje rozhraní ArrayAccess a Countable.